examples/ActionGroup/ActionGroup.Tests.ps1

```diff
@@ -2,31 +2,69 @@ BeforeAll {
   Import-Module Az.InfrastructureTesting
 }
 
+$resourceType = "ActionGroup"
+$resourceName = "sampleaction"
+$rgName = "rg-test"
+
+Describe 'Verify Action Group with Confirm-AzBPResource' {
+  it 'Should contain an actiongroup named sampleaction' {
+    #arrange
+    $params = @{
+      ResourceType      = $resourceType
+      ResourceName      = $resourceName
+      ResourceGroupName = $rgName
+    }
+
+    #act
+    $result = Confirm-AzBPResource @params
+
+    #assert
+    $result.Success | Should -Be $true
+  }
+
+  it 'Should contain an actiongroup named sampleaction' {
+    #arrange
+    $params = @{
+      ResourceType      = $resourceType
+      ResourceName      = $resourceName
+      ResourceGroupName = $rgName
+      PropertyKey       = "GroupShortName"
+      PropertyValue     = $resourceName
+    }
+
+    #act
+    $result = Confirm-AzBPResource @params
+
+    #assert
+    $result.Success | Should -Be $true
+  }
+}
+
 Describe 'Verify Action Group Exists' {
-  it 'Should contain a action group named sampleaction' {
+  it 'Should contain an action group named sampleaction' {
     #arrange
-    $resourceGroupName = "rg-test"
-    $actionGroupName = "sampleaction"
+    $rgName = $rgName
```

Comment:
There's no need for any of these tests to have assignments, the variables that you introduced at the top of the file can be used directly (here and in the other `Test` files).

This would not apply to the negative test's failure value, of course, unless you choose to set that at the top as well.

Reply:
Do we need `#arrange` if there is no step for some of these? Deleted mine